Library/Formula/cloc.rb

```diff
@@ -3,11 +3,18 @@ class Cloc < Formula
   homepage "https://github.com/AlDanial/cloc/"
   url "https://github.com/AlDanial/cloc/releases/download/1.64/cloc-1.64.pl"
```

Comment:
Use https://github.com/AlDanial/cloc/archive/1.64.tar.gz as the URL and then you can have the same logic for stable and head.